tests/IndexTest.php

```diff
@@ -14,4 +14,11 @@ public function testIndex() {
              ->see('>Sign In</a>') // Ensure log in buttons are shown when user is logged out
              ->dontSee('SQLSTATE'); // Ensure database connection is correct
     }
+    
+    public function testRequestGetNotExistShortUrl() {
```

Comment:
Could we move this to a different test? IndexTest seems mainly dedicated to testing the main page :)
Maybe `LinkControllerTest.php`